lib/diagnostic.js

```diff
@@ -6,29 +6,56 @@
 // splits it into an array of normalized words
 // (uppercase strings without punctuation)
 // and returns that array.
-const normalizeWords = function (/* */) {
-  /* your response here */
+const normalizeWords = function (str) {
+  let words = str.split(" ");
```

Comment:
This should be split using the regular expression ```/\s+/``` as its argument rather than a single space.  ```/\s+/``` matches all whitespace (spaces, tabs, line breaks, etc.).